src/main/java/io/confluent/connect/jdbc/TimestampIncrementingTableQuerier.java

```diff
@@ -147,18 +149,25 @@ protected void createPreparedStatement(Connection db) throws SQLException {
   @Override
   protected ResultSet executeQuery() throws SQLException {
     if (incrementingColumn != null && timestampColumn != null) {
-      Timestamp ts = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
-      stmt.setTimestamp(1, ts, UTC_CALENDAR);
-      stmt.setLong(2, (incrementingOffset == null ? -1 : incrementingOffset));
-      stmt.setTimestamp(3, ts, UTC_CALENDAR);
-      log.debug("Executing prepared statement with timestamp value = " + timestampOffset + " and incrementing value = " + incrementingOffset);
+      Timestamp startTime = new Timestamp(timestampOffset == null ? 0 : timestampOffset);
+      Timestamp endTime = new Timestamp(new Date().getTime() - timestampDelay);
```

Comment:
That was my first attempt too.
I failed to find syntax that works across the board, but will take another look there.

Any thoughts on whether we should also support "delayed ID"s? 
for similar reasons, transactions with newer sequences can finish later. On the other hand, unlike time, IDs don't move ahead uniformly so it can lead to some weird conditions.